changelogs/head.asciidoc

```diff
@@ -27,3 +28,5 @@ https://github.com/elastic/apm-server/compare/8.6\...main[View commits]
 - Add `event.success_count` to transaction events and transaction metrics {pull}9819[9819]
 - Dedicated overflow buckets for transaction and service aggregation to limit cardinality {pull}9856[9856]
 - Automatically scale `MaxGroups` and `MaxTransactionGroups` based on available memory {pull}9856[9856]
+- Improve APM UI query performance by introducing `10m` and `60m` aggregation intervals for transaction metrics. Store them into dedicated data streams `metrics-apm.transaction.${interval}` {pull}9846[9846]
+- Improve APM UI query performance by introducing `10m` and `60m` aggregation intervals for span destination metrics. Store them into dedicated data streams `metrics-apm.span_destination.${interval}` {pull}9926[9926]
```

Comment:
I would read the changelog entries that there is no `1m` interval anymore, but that's not true; can you please rephrase a bit?